### ISSUE TITLE
Input with Value on Hidden Element Label Overlap

### ISSUE THREAD

**Issue by cryptanym** [beercss/beercss#154]
Inputs that have values on hidden elements such as tab pages, will not have their label upated on the page loading, in this codepen you can see it works correctly for Tab 1 on page load, but not Tab 2 and  Tab 3.

https://codepen.io/1971562171/pen/rNqqOLE

I have different account options in tabs, which is filled with the current options, which has this overlap show up, and is quite visually unappealing when you have lots of inputs, otherwise it's a small issue.

**Comment by cryptanym** [comment]
Oh great thanks i'll try that now!

**Comment by KicKerBNU** [comment]
I'm not sure but i think this happen because the changes are made in the Virtual DOM and Beercss dont have access to this changes, correct me if i'm wrong @leonardorafael

**Comment by leonardorafael** [comment]
@KicKerBNU this seems a big mistery to me. Maybe a little delay resolves. We already have an MutationObserver that detects changes on DOM (it's very lightweight, but not detect all changes, only add/remove elements).

**Comment by cryptanym** [comment]
I updated the CodePen with simple JavaScript to fix the issue for me, https://codepen.io/1971562171/pen/rNqqOLE

The timeout seems to make it work better.

**Comment by KicKerBNU** [comment]
For the codepen example just an ui() resolves the problem, im happy this works for you 👍 

```
if (element.tagName.toLowerCase() === 'a' && parentElement.classList.contains('tabs'))  ui();
```

**Comment by leonardorafael** [comment]
We found the bug. I hope we don't need to call `ui()` anymore at this time. Fixed in v3.2.4.

We created a codepen using the new version here https://codepen.io/leo-bnu/pen/WNaaBER
